locales/de.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Read more": "Lies mehr",
+  "Featured": "Hervorgehoben",
+  "Powered by {ghostlink}": "Läuft mit {ghostlink}",
+  "Search posts, tags and authors": "Suche nach Beiträgen, Tags und Autoren",
+  "Subscribe": "Abonnieren",
+  "Sign in": "Anmelden",
+  "Account": "Account",
```

Comment:
Wird hier mit "Konto" übersetzt: https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/locales/de/portal.json#L13

Reply:
and "Sign in" is translated "Einloggen" here: https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/locales/de/portal.json#L105 . might be good to use same translations.